projects/Bactteria_Dataset/README.md

```diff
@@ -0,0 +1,110 @@
+# Bactteria detection with darkfield microscopy Dataset
+
+This project support **`Bactteria detection with darkfield microscopy Dataset`**, and the dataset can be download from [here](https://tianchi.aliyun.com/dataset/94411).
+
+## Usage
+
+<!-- For a typical model, this section should contain the commands for training and testing. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
+
+### Prerequisites
+
+- Python 3.7
+- PyTorch 1.6 or higher
+- [MIM](https://github.com/open-mmlab/mim) v0.33 or higher
+- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation) v1.0.0rc2 or higher
```

Comment:
Should add mmcv version, and as versions of mmcv and mmsegmentation have compatibility issues, I suggest fixxing the versions of all packages. Please refer to https://github.com/open-mmlab/mmsegmentation/blob/1.x/docs/en/notes/faq.md#installation for more details